proto/core.proto

```diff
@@ -66,5 +67,7 @@ message Msg {
     Command cmd = 1;
     ErrorCmd error = 2;
     AppVersionCmd app_version = 3;
+    SessionStartCmd session_start = 4;
+    SessionCloseCmd session_close = 5;
   }
```

Comment:
there is build issue after added the Session Cmd for CORE_MSG_SIZE

Reply:
fixed